source/opt/spread_volatile_semantics.cpp

```diff
@@ -142,6 +116,8 @@ bool SpreadVolatileSemantics::IsTargetUsedByNonVolatileLoadInEntryPoint(
     uint32_t var_id, Instruction* entry_point) {
   uint32_t entry_function_id =
       entry_point->GetSingleWordInOperand(kOpEntryPointInOperandEntryPoint);
+  std::unordered_set<uint32_t> funcs;
+  context()->CollectCallTreeFromRoots(entry_function_id, &funcs);
   return !VisitLoadsOfPointersToVariableInEntries(
```

Comment:
Could you please update the documentation for `SpreadVolatileSemantics::VisitLoadsOfPointersToVariableInEntries`?  All of the documentation for it says that the function ids are entry points, but you are not passing in other functions.  The function definition does not have to change.